testsuite/MDAnalysisTests/test_atomselections.py

```diff
@@ -606,6 +606,32 @@ def setUp(self):
     def tearDown(self):
         del self.u
 
+    def _check_cyzone(self, meth, periodic):
```

Comment:
Yeah, I'd rather keep the same pattern as all the other tests generators in the class